docs/docusaurus/docs/core/set_up_a_gx_environment/create_a_data_context.md

```diff
@@ -18,11 +18,11 @@ All scripts that utilize GX Core should start with the creation of a Data Contex
 
 The following are the available Data Context types:
 
-- **File Data Context:** A persistent Data Context that stores metadata and configuration information as YAML files.
+- **File Data Context:** A persistent Data Context that stores metadata and configuration information as YAML files. File Data Contexts allow you to re-use previously configured Expectation Suites, Data Sources, and Checkpoints.
```

Comment:
This addition is kind of redundant, but given historical user understanding of how to select a Data Context type, I think it might be helpful to be overly explicit that the File Data Context stores configurations & metadata on disk. Feel free to tweak the wording if you want to keep the intent of the change but have a more elegant way to express it. :)